admin/controllers/school.js

```diff
@@ -458,6 +471,13 @@ const getSelectPupilNotTakingCheck = async (req, res, next) => {
   })
 }
 
+/**
```

Comment:
where are the unit tests for this controller?

Reply:
In addition to guys comment you need to also test the new service function

Reply:
@GuyHarwood @GeorgeChatzigiannis As explained and agreed, merging as it is comes first, while signing off happens I'm already in the process of simplifying controllers, writing services all using a TDD approach - so refactoring and unit tests come in the next PR.